action.yml

```diff
@@ -5,7 +5,9 @@ inputs:
   build_arch:
     description: What arch to build for. Comma seperated list
     default: linux/amd64
-    required: true
+  dockerfile:
```

Comment:
Perhaps `filename`?

Reply:
the original `file` was a bit to fuzzy for me. shall we better go with filename? but to be more precize one could argue the use dockerfile_filename 🤔

Reply:
I was debating this, but it was merged before I could respond. I was also questioning the default and wondering if the original action also considers `Containerfile`.

Reply:
the wrapped action has also a default for `Dockerfile`. One could also specify Containerfile now. But the original is only build for docker. But the spec ist the same(?) so this should work. i can open a new PR and rename the parameter or we could move the conversation over there?

Reply:
I don't feel too strongly about it, but in general I'd like to see the world embrace `Containerfile` instead of `Dockerfile` since it's not branded. So small things like this can help with that.